src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  flex-grow: 1;
+  flex-shrink: 0;
+
+  height: 300px;
+  flex-basis: 33%;
+
+  justify-content: center;
+  align-items: center;
+
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+}
+
+.box--1 {
+  background-color: red;
+}
+
+.box--2 {
+  background-color: rgb(204, 0, 0);
+}
```

Comment:
be consistent with this. do not use both RGB and color names. I'd only use RGB or hex because it's cooler